README.md

```diff
@@ -115,12 +115,15 @@ This will allow you to build the *exact* same TCs used to build the nightlies, t
 
 On Mac OS X you may need to install the following tools using [Homebrew](https://brew.sh/):
 ```
-brew install nasm binutils libtool autoconf automake cmake makedepend sdl2 lua51 gettext pkg-config wget md5sha1sum
+brew install nasm binutils libtool autoconf automake cmake makedepend sdl2 lua@5.1 luarocks gettext pkg-config wget md5sha1sum
 echo 'export PATH="/usr/local/opt/gettext/bin:$PATH"' >> "$HOME"/.bash_profile
 ```
 
 If you run into a gettext error while building glib, try `brew link --force gettext` to override the built-in Mac OS BSD gettext with GNU GetText.
 
+*Note:* in Mojave you need to set a minimum deployment version higher than 10.04. Otherwise you'll get the error `ld: library not found for -lgcc_s.10.4`
+`export MACOSX_DEPLOYMENT_TARGET=10.10`
+
 The KOReader Android build requires `ant`, `openjdk-8-jdk` and `p7zip-full`. A compatible version of the Android NDK and SDK will be downloaded automatically by `./kodev build android` if no NDK or SDK is provided in environment variables. For that purpose you can use `NDK=/ndk/location SDK=/sdk/location ./kodev build android`.
 
 Users of Debian Jessie first need to configure the `backports` repository:
```

Comment:
Oh, I guess that a little further down I should add a note about how in Debubuntu you can use `update-alternatives`. See https://github.com/koreader/koreader/issues/4254#issuecomment-440025378 where I just shared that as some kind of common knowledge, but I'm not sure how long it took before I learned about it. :-)

Reply:
👍 Will do it in other PR with some building documentation refactor.